prep-css-syntax/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CSS Syntax</title>
+</head>
+  <style>
+  h1 {font-family: Arial, Helvetica, sans-serif;}
+  p {font-size: 2rem;}
+  h3 {color: firebrick;}
+  h2 { font-style: italic;}
+</style>
+
+<style>
+  h1 {font-family: Arial, Helvetica, sans-serif;}
+  p {font-size: 2rem;}
+  h3 {color: firebrick;}
+  h2 { font-style: italic;}
+</style><body>
```

Comment:
You can remove one of these `<style>` elements and be sure to keep `<style>` inside of the `<head>` of the document.

Reply:
edited!

Reply:
![CSS Assignment 1 Edited](https://user-images.githubusercontent.com/97069037/149217244-f1ad9776-ff8f-4d34-91db-1f1414998cfa.png)